protocols/v2/codec-sv2/src/encoder.rs

```diff
@@ -1,3 +1,47 @@
+//! # Stratum V2 Codec Encoder
+//!
+//! This module provides functionality for encoding Stratum V2 messages, including support for the
+//! Noise protocol for secure communication.
```

Comment:
https://github.com/stratum-mining/stratum/pull/1111#pullrequestreview-2248938300

```suggestion
//! Provides functionality for encoding Stratum V2 messages, including support for the
//! Noise protocol for secure communication.
```